www/lynx/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 
 name                lynx
-set milestone       2.8.9
-version             ${milestone}rel.1
+set milestone       2.9.0
+version             ${milestone}dev.11
```

Comment:
this name suggest it's actually not a released version but a development version. Typically one would only update a port to its latest, stable release.